fix_summary_bloc_test.py

```
@@ -0,0 +1,179 @@
+with open('test/features/analytics/presentation/bloc/summary_bloc_test.dart', 'r') as f:
```

Ruff (0.15.2)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)